R/methods-epi_archive.R

```diff
@@ -941,26 +785,52 @@ group_by.epi_archive <- function(.data, ..., .add = FALSE, .drop = dplyr::group_
 #'   filter(geo_value == "ca") %>%
 #'   select(-geo_value)
 #'
-#' @importFrom rlang enquo !!!
 #' @export
-epix_slide <- function(x, f, ..., before, ref_time_values,
-                       time_step, new_col_name = "slide_value",
-                       as_list_col = FALSE, names_sep = "_",
-                       all_versions = FALSE) {
+epix_slide <- function(
+    x,
+    f,
+    ...,
+    before,
+    ref_time_values,
+    time_step,
+    new_col_name = "slide_value",
+    as_list_col = FALSE,
+    names_sep = "_",
+    all_versions = FALSE) {
   if (!is_epi_archive(x, grouped_okay = TRUE)) {
     cli_abort("`x` must be of class `epi_archive` or `grouped_epi_archive`.")
   }
```

Comment:
note: this type check is unnecessary as S3 dispatch will take care of it for us.  Is this for a clearer error message than S3 failed dispatch?

Reply:
Nah, just left-over from when epix_slide did not S3 dispatch https://github.com/cmu-delphi/epiprocess/blob/4e65e51bb56ab70cc98fa2d37dd35a4ab2336620/R/methods-epi_archive.R#L946-L962. Removing this.